Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -410,3 +417,8 @@
 "history" = "history";
 "settings" = "Settings";
 "help" = "help";
+
+/* PIN Setup*/
+"next" = "Next";
+"create" = "Create";
```

Comment:
Should be in the `Request PIN and PIN Setup` section. Same for other translation files.